src/benchmark/trulens/benchmark/benchmark_frameworks/experiments/rag_triad_baselines.ipynb

```diff
@@ -23,241 +23,29 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    ms_marco_hard_neg_balanced = pd.read_csv("data/ms_marco_hard_neg_balanced.csv")
</pre><p>can we show the BEIR loader here instead of reading from local?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1559/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
`ms_marco_hard_neg_balanced` is not available in BEIR. It's a distinct dataset https://huggingface.co/datasets/nixiesearch/ms-marco-hard-negatives